public/app/features/explore/LogsContainer.tsx

```diff
@@ -104,66 +103,60 @@ class LogsContainer extends PureComponent<LogsContainerProps> {
       return null;
     }
 
-    // We need to override css overflow of divs in Collapse element to enable sticky Logs navigation
-    const styleOverridesForStickyNavigation = css`
-      & > div {
-        overflow: visible;
-        & > div {
-          overflow: visible;
-        }
-      }
-    `;
-
     return (
       <>
         <LogsCrossFadeTransition visible={isLive}>
-          <Collapse label="Logs" loading={false} isOpen>
-            <LiveTailControls exploreId={exploreId}>
-              {(controls) => (
-                <LiveLogsWithTheme
-                  logRows={logRows}
-                  timeZone={timeZone}
-                  stopLive={controls.stop}
-                  isPaused={this.props.isPaused}
-                  onPause={controls.pause}
-                  onResume={controls.resume}
-                />
-              )}
-            </LiveTailControls>
-          </Collapse>
+          <>
+            {/* this empty-div takes the place of the logs-volume-histogram in the crossfade transition animation */}
+            <div />
+            <Collapse label="Logs" loading={false} isOpen>
+              <LiveTailControls exploreId={exploreId}>
+                {(controls) => (
+                  <LiveLogsWithTheme
+                    logRows={logRows}
+                    timeZone={timeZone}
+                    stopLive={controls.stop}
+                    isPaused={this.props.isPaused}
+                    onPause={controls.pause}
+                    onResume={controls.resume}
+                  />
+                )}
+              </LiveTailControls>
+            </Collapse>
+          </>
```

Comment:
```suggestion
          <Collapse label="Logs" loading={false} isOpen>
            <LiveTailControls exploreId={exploreId}>
              {(controls) => (
                <LiveLogsWithTheme
                  logRows={logRows}
                  timeZone={timeZone}
                  stopLive={controls.stop}
                  isPaused={this.props.isPaused}
                  onPause={controls.pause}
                  onResume={controls.resume}
                />
              )}
            </LiveTailControls>
          </Collapse>
```

i believe this is a mistake? without the extra div and the fragment the animation looks ok :

https://user-images.githubusercontent.com/1170767/189865053-2aea67f8-34f0-478c-b3b3-d85d90a8c3ec.mov

Reply:
committed 👍